composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "php": "^7.4|^8.0",
         "spatie/laravel-package-tools": "^1.1",
-        "illuminate/contracts": "^8.0"
+        "illuminate/contracts": "^8.0",
```

Comment:
Hi, since the `"illuminate/contracts"` is called on the `"tipoff/support": "^1.0"` you can remove from here.